isic_kredit/Info.plist

```diff
@@ -16,6 +18,17 @@
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>isic_konto</string>
```

Comment:
K čemu potřebujem URL schéma? 🤔

Reply:
To jsem zkousel nejak udelat ten TodayExtension ale uz je to pryc